app/Console/Kernel.php

```diff
@@ -43,6 +44,8 @@ protected function schedule(Schedule $schedule)
         $schedule->command('schedule:stats')->dailyAt('7:00');
 
         $schedule->job(new PollForMediaWikiJobsJob)->everyFifteenMinutes();
+
+        $schedule->job(new UpdateWikiSiteStatsJob)->dailyAt('19:00');
```

Comment:
This is totally arbitrary.

Reply:
We just recently moved another job _off_ the midnight schedule to reduce load, so I would think it's wise not to run this at midnight as well.

https://phabricator.wikimedia.org/T317980

I'd suggest keeping this at 19:00. Considering we have users around the globe, it's also not really possible to tell when people are sleeping or not.

Reply:
I am not aware how many jobs we run during the night. But if you said so, let's just keep 19:00

Reply:
The Logical SQL backup (which is not a Laravel) job is running at Midnight and is pretty expensive, so it's best not to schedule anything else during that period https://github.com/wmde/wbaas-deploy/blob/b98f2fcf235a7e56ac1a8b889fc7a13f04be6df8/k8s/helmfile/env/production/wbaas-backup.values.yaml.gotmpl#L6-L7